language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/command/CommandExecutionTest.java

```diff
@@ -150,17 +161,138 @@ public void testCreateVariable(String config, String source) {
         JsonObject configJsonObject = FileUtils.fileContentAsObject(configJsonPath);
         JsonObject expected = configJsonObject.get("expected").getAsJsonObject();
         List<Object> args = new ArrayList<>();
-        args.add(new CommandUtil.CommandArgument(CommandConstants.ARG_KEY_DOC_URI, sourcePath.toUri().toString()));
-        args.add(new CommandUtil.CommandArgument(CommandConstants.ARG_KEY_FUNC_LOCATION,
-                                                 configJsonObject.get("functionLocation").getAsString()));
-        args.add(new CommandUtil.CommandArgument(CommandConstants.ARG_KEY_RETURN_TYPE,
-                                                 configJsonObject.get("functionReturnType").getAsString()));
-        args.add(new CommandUtil.CommandArgument(CommandConstants.ARG_KEY_VAR_NAME,
-                                                 configJsonObject.get("variableName").getAsString()));
+        JsonObject arguments = configJsonObject.get("arguments").getAsJsonObject();
+        args.add(new CommandArgument(CommandConstants.ARG_KEY_DOC_URI, sourcePath.toUri().toString()));
+        args.add(new CommandArgument(CommandConstants.ARG_KEY_NODE_LINE, arguments.get("node.line").getAsString()));
+        args.add(new CommandArgument(CommandConstants.ARG_KEY_NODE_COLUMN, arguments.get("node.column").getAsString()));
         JsonObject responseJson = getCommandResponse(args, CommandConstants.CMD_CREATE_VARIABLE);
         responseJson.get("result").getAsJsonObject().get("edit").getAsJsonObject().getAsJsonArray("documentChanges")
                 .forEach(element -> element.getAsJsonObject().remove("textDocument"));
-        Assert.assertTrue(responseJson.equals(expected));
+        Assert.assertEquals(responseJson, expected);
+    }
+
+    @Test(dataProvider = "testgen-fail-data-provider")
+    public void testTestGenerationFailCases(String config, Path source) throws IOException {
+        String configJsonPath = "command" + File.separator + config;
+        Path sourcePath = sourcesPath.resolve("source").resolve(source);
+        Path testFilePath = LSCompilerUtil.getCurrentModulePath(sourcePath).resolve("tests")
```

Comment:
Here instead of using "tests" as a string shall we use the constant "ProjectDirConstants.TEST_DIR_NAME"?